Gradients/Sources/Gradients+Layer.swift

```diff
@@ -10,492 +10,494 @@ public extension Gradients {
+            return Gradients.linear(to: .top, colors: [0xf3e7e9, 0xe3eeff, 0xe3eeff], locations: [0.0, 0.99, 1.0])
         case .bigMango:
-            return linear(to: .top, colors: [0xc71d6f, 0xd09693], locations: [0.0, 1.0])
+            return Gradients.linear(to: .top, colors: [0xc71d6f, 0xd09693], locations: [0.0, 1.0])
         case .healthyWater:
-            return linear(to: .degree(60), colors: [0x96deda, 0x50c9c3], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(60), colors: [0x96deda, 0x50c9c3], locations: [0.0, 1.0])
         case .amourAmour:
-            return linear(to: .top, colors: [0xf77062, 0xfe5196], locations: [0.0, 1.0])
+            return Gradients.linear(to: .top, colors: [0xf77062, 0xfe5196], locations: [0.0, 1.0])
         case .riskyConcrete:
-            return linear(to: .top, colors: [0xc4c5c7, 0xdcdddf, 0xebebeb], locations: [0.0, 0.52, 1.0])
+            return Gradients.linear(to: .top, colors: [0xc4c5c7, 0xdcdddf, 0xebebeb], locations: [0.0, 0.52, 1.0])
         case .strongStick:
-            return linear(to: .right, colors: [0xa8caba, 0x5d4157], locations: [0.0, 1.0])
+            return Gradients.linear(to: .right, colors: [0xa8caba, 0x5d4157], locations: [0.0, 1.0])
         case .viciousStance:
-            return linear(to: .degree(60), colors: [0x29323c, 0x485563], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(60), colors: [0x29323c, 0x485563], locations: [0.0, 1.0])
         case .paloAlto:
-            return linear(to: .degree(-60), colors: [0x16a085, 0xf4d03f], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(-60), colors: [0x16a085, 0xf4d03f], locations: [0.0, 1.0])
         case .happyMemories:
-            return linear(to: .degree(-60), colors: [0xff5858, 0xf09819], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(-60), colors: [0xff5858, 0xf09819], locations: [0.0, 1.0])
         case .midnightBloom:
-            return linear(to: .degree(-20), colors: [0x2b5876, 0x4e4376], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(-20), colors: [0x2b5876, 0x4e4376], locations: [0.0, 1.0])
         case .crystalline:
-            return linear(to: .degree(-20), colors: [0x00cdac, 0x8ddad5], locations: [0.0, 1.0])
+            return Gradients.linear(to: .degree(-20), colors: [0x00cdac, 0x8ddad5], locations: [0.0, 1.0])
         case .raccoonBack:
             let layer = CALayer()
-            layer.addSublayer(linear(to: .degree(180), colors: [0xBCC5CE, 0x929EAD], locations: [0.0, 1.0]))
-            layer.addSublayer(radial(startPoint: CGPoint(x: 0.0, y: 0.0),
+            layer.addSublayer(Gradients.linear(to: .degree(180), colors: [0xBCC5CE, 0x929EAD], locations: [0.0, 1.0]))
+            layer.addSublayer(Gradients.radial(startPoint: CGPoint(x: 0.0, y: 0.0),
                                      endPoint: CGPoint(x: 1.0, y: 1.0),
```

Comment:
@muhasturk thank you for your second PR 👻 

Would you please align indentation about creating radial gradient?